2_Bind_and_clean_seqs.R

```diff
@@ -0,0 +1,66 @@
+## Emily Hodgson
+## Last modified: 3.11.2023
+
+# Setup
+package.setup <- function() {
```

Comment:
The script should not include package setup (installing/loading packages). There should be a separate script for this: see the RSetup.R script.